Changed default server for samples from IIS Express to Kestrel

ShreyasJejurkar opened the pull request:
Keeping kestrel as the first option, VS will automatically show as the default option selected for debugging! (If not, make sure you delete the `.vs` folder from the root of the cloned repo. )

I have kept some IIS servers test project `launchSettings.json` untouched because keeping IIS Express as the default option for them makes sense! 

Fixes https://github.com/dotnet/aspnetcore/issues/27630

ShreyasJejurkar committed:
Changed default server for samples from IIS Express to Kestrel

Fixes https://github.com/dotnet/aspnetcore/issues/27630

javiercn commented:
@MCCshreyas thanks for your contribution!

It looks great, the only comment I have is that I believe there might be missing projects that need this treatment, for example the ones in Blazor. Ideally we want to tackle all the projects in the repo in one go.

@Tratcher can you look at the ones in Server?

MCCshreyas commented:
Sure @javiercn I will get those addressed in this PR only!

ShreyasJejurkar committed:
Addressed remaining projects excluding IIS specific tests projects!

Tratcher reviewed (commented):
The three projects under `/src/Servers/HttpSys/samples/` (except TestClient) should also have a launchsettings that only includes the Project profile, no IIS.

MCCshreyas commented on src/DefaultBuilder/testassets/CreateDefaultBuilderApp/Properties/launchSettings.json:
```diff
@@ -0,0 +1,40 @@
+{
```
I did not added any `launchSettings.json` explicitly! I will check this!

MCCshreyas replied:
@Tratcher It was already there! I did not added that explicitly!

MCCshreyas commented on src/Servers/Kestrel/samples/Http3SampleApp/Properties/launchSettings.json:
```diff
@@ -0,0 +1,27 @@
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    },
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    }
```
OK, thanks. I will remove the IIS config directly!

Intension of PR was to give kestrel as default option, but as per your suggestion for this specific projects I will remove IIS config and keep kestrel only!

MCCshreyas commented on src/Servers/IIS/IIS/test/IIS.Tests/Properties/launchSettings.json:
```diff
@@ -0,0 +1,40 @@
+{
```
It was already there, but as per your suggestion, I deleted the `launchSettings.json`

MCCshreyas commented on src/Servers/testassets/ServerComparison.TestSites/Properties/launchSettings.json:
```diff
@@ -0,0 +1,40 @@
+{
```
It was there already. Now converted to project profile!

ShreyasJejurkar committed:
Addressed PR feedback.

Removed IIS Settings from launchSettings and added launchSettings.json for HttpSys samples projects

Tratcher commented:
Thanks
